### PR TITLE
add ipydagred3

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by timkpaine** [comment]
https://github.com/conda-forge/staged-recipes/pull/11431

File: recipes/ipydagred3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - nodejs
+    - jupyter-packaging
```
Comment (hmaarrfk) [inline recipes/ipydagred3/meta.yaml:22]:
```suggestion
    - jupyter-packaging
    - setuptools
```

File: recipes/ipydagred3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a1acc34f25909a1fa5d5bc89c8fcf7874a968fcefb71a2f80cb2c0ef22eca6af
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (hmaarrfk) [inline recipes/ipydagred3/meta.yaml:15]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv --no-build-isolation"
```
you shouldn't need the above tag, but maybe

**Comment by timkpaine** [comment]
@conda-forge/help-python all set!

**Comment by timkpaine** [comment]
@hmaarrfk / @conda-forge/help-python can you merge, need this for tributary

thanks!

**Comment by nehaljwani** [comment]
Thank you @timkpaine @hmaarrfk !

**Pull request merged**